src/views/CountryRiskWatch/RiskTable/index.tsx

```diff
@@ -0,0 +1,257 @@
+type CountryRiskResponse = GetCountryRisk['responses']['200']['content']['application/json'];
+type RiskData = CountryRiskResponse[number];
+
+interface Props {
+    className?: string;
+    riskData: RiskData | undefined;
+    selectedMonths: Record<number, boolean> | undefined;
+    dataPending: boolean;
+}
+
+function RiskTable(props: Props) {
+    const {
+        riskData,
+        className,
+        selectedMonths,
+        dataPending,
+    } = props;
+
+    const strings = useTranslation(i18n);
+
+    const fiData = useMemo(
+        () => getFiRiskDataItem(riskData?.ipc_displacement_data),
+        [riskData],
+    );
+
+    const wfData = useMemo(
+        () => getWfRiskDataItem(riskData?.gwis),
+        [riskData],
+    );
+
+    const hazardTypeList = useMemo(
+        () => (
+            unique(
+                [
+                    ...riskData?.idmc?.filter(hasSomeDefinedValue) ?? [],
+                    ...riskData?.raster_displacement_data?.filter(hasSomeDefinedValue) ?? [],
+                    ...riskData?.inform_seasonal?.filter(hasSomeDefinedValue) ?? [],
+                    fiData,
+                    wfData,
+                ].filter(isDefined).map(getDataWithTruthyHazardType).filter(isDefined),
+                (data) => data.hazard_type,
+            ).map((combinedData) => ({
+                hazard_type: combinedData.hazard_type,
+                hazard_type_display: combinedData.hazard_type_display,
+            }))
+        ),
+        [riskData, fiData, wfData],
+    );
+
+    type HazardTypeOption = (typeof hazardTypeList)[number];
+
+    const riskScoreToCategory = useCallback(
+        (score: number | undefined | null, hazardType: HazardTypeOption['hazard_type']) => {
+            if (isNotDefined(score) || score < 0) {
+                return '-';
+            }
+
+            const scoreCategories = hazardType === 'WF' ? [
+                { score: 1000, label: strings.riskScoreVeryHighLabel },
+                { score: 17, label: strings.riskScoreHighLabel },
+                { score: 9, label: strings.riskScoreMediumLabel },
+                { score: 5, label: strings.riskScoreLowLabel },
+                { score: 2, label: strings.riskScoreVeryLowLabel },
+            ] : [
+                { score: 10, label: strings.riskScoreVeryHighLabel },
+                { score: 6.5, label: strings.riskScoreHighLabel },
+                { score: 5, label: strings.riskScoreMediumLabel },
+                { score: 3.5, label: strings.riskScoreLowLabel },
+                { score: 2, label: strings.riskScoreVeryLowLabel },
+            ];
+
+            const currentCategory = scoreCategories.reverse().filter(
+                (category) => score <= category.score,
+            );
+
+            if (currentCategory.length === 0) {
+                return strings.riskScoreNotApplicableLabel;
+            }
+
+            return currentCategory[0].label;
```

Comment:
Why are we taking the first element? Why are we using reverse?

Reply:
We're trying to find the category for the selected score

Reply:
Let's discuss later.